bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -60,6 +60,7 @@ enum OperationType {
     WRITE_LAC = 6;
     READ_LAC = 7;
     GET_BOOKIE_INFO = 8;
+    STARTTLS = 9;
```

Comment:
To follow the convention of the previous operations, should we call it `START_TLS`?